openslides/mediafiles/static/js/mediafiles/image-plugin.js

```diff
@@ -0,0 +1,202 @@
+(function () {
+
+'use strict';
+
+angular.module('OpenSlidesApp.mediafiles.image-plugin', [])
+
+.factory('ImageBrowserPlugin', [
+    '$http',
```

Comment:
What do you exactly mean by order? Do we have a preferred order? I always do the angular staff first, then the resources and then all other stuff needed.